.github/workflows/e2e-test.yml

```diff
@@ -89,8 +92,40 @@ jobs:
           cd RhinoApp
           Rscript ../test-build-sass.R
 
+      - name: test_r() should run testthat tests
+        if: always()
+        run: |
+          cd RhinoApp
+          Rscript -e "rhino::test_r()"
+

```

Comment:
This works now because of `checkout-files`. I did not do anything standalone for this step because the full test later should confirm that `rhino::test_r()` works. This only confirms that the base best , `test-main()` and the function, works.